src/components/Dropdown/Dropdown.md

```diff
@@ -0,0 +1,50 @@
+Use the dropdown component to create a list of menu options that can be toggled with a button element.
+
+View the [Rivet documentation for Dropdowns](https://rivet.iu.edu/components/dropdown/).
+
+### Dropdown Example
+
+Use the `label` property to set the text on the dropdown button.
```

Comment:
We might either tweak the wording here or add another example to show that React nodes can be used for labels as well.

Reply:
Added an example.